src/components/Patient/PatientIdentifierSelector.tsx

```
@@ -0,0 +1,509 @@
+              >
+                <QrCode className="size-4 mr-1" />
+                {t("scan")}
+              </Button>
+            </div>
+
+            {/* Search Type Selector */}
+            {allIdentifierConfigs.length > 2 ? (
+              <div>
+                <label className="text-xs text-gray-600 mb-1.5 ml-1 block">
+                  {t("search_by")}
+                </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)